packages/core/src/components/button/abstractButton.tsx

```diff
@@ -22,6 +22,14 @@ export interface IButtonProps extends IActionProps {
      */
     active?: boolean;
 
+    /**
+     * Text alignment within button. By default, icons and text will be centered within the button.
+     * Passing this prop will cause the text container to fill the button and align the text within that
+     * to the appropriate side. `icon` and `rightIcon` will be pushed to either side.
+     * @default "center"
+     */
+    align?: "left" | "center" | "right";
```

Comment:
😍 😍 😍

Reply:
this conflicts somewhat with the `NavbarGroup` align prop -- here, you are referring to the alignment of _content_ while for NavbarGroup (as well as most design tools), `align` refers to the component's alignment in its container. I suggest renaming this to `textAlignment` or `alignText`

Reply:
mmm yes good catch @adidahiya! i'll go with `alignText`